ocs_ci/framework/conf/default_config.yaml

```diff
@@ -101,12 +101,12 @@ ENV_DATA:
   worker_replicas: 3
   skip_ocp_deployment: false
   skip_ocs_deployment: false
-  ocs_version: '4.4'
+  ocs_version: '4.5'
   # uncomment to use custom directory for storing measurement data related to
   # monitoring tests, otherwise generate temporary directory for each test run
   # measurement_dir: '/tmp/ocs_ci_monitoring_measurement/'
   # Default RHCOS image to be used for VmWare deployment
-  vm_template: 'rhcos-4.4.0-rc.1-x86_64-vmware.x86_64'
+  vm_template: 'rhcos-4.5.0-rc.1-x86_64-vmware.x86_64'
```

Comment:
```suggestion
  vm_template: ' rhcos-4.5.0-0.nightly-2020-03-18-115438-x86_64-vmware.x86_64'
```

Reply:
@vavuthu is the space at the start of the template string necessary or a typo?

Reply:
i think just a typo

Reply:
I updated the vm_template without the space at the start, assuming it was a typo.

Reply:
its typo